storm-core/src/jvm/backtype/storm/serialization/SerializationFactory.java

```diff
@@ -81,8 +81,9 @@ public static Kryo getKryo(Map conf) {
         kryoFactory.preRegister(k, conf);
 
         boolean skipMissing = (Boolean) conf.get(Config.TOPOLOGY_SKIP_MISSING_KRYO_REGISTRATIONS);
-        for(String klassName: registrations.keySet()) {
-            String serializerClassName = registrations.get(klassName);
+        for(Map.Entry<String, String> entry: registrations.entrySet()) {
+            String klassName = entry.getKey();
```

Comment:
There is another variable of type Class klass that is used in that part of the code. So the name klassName is consistent with it. Any way the next version of the patch just uses entry.getKey() directly.